android/src/main/java/com/asterinet/react/bgactions/BackgroundTaskOptions.java

```diff
@@ -68,6 +75,25 @@ public BackgroundTaskOptions(@NonNull final ReactContext reactContext, @NonNull
         } catch (Exception e) {
             extras.putInt("color", Color.parseColor("#ffffff"));
         }
+
+        // Get actions
+        try {
+            final ReadableArray acts = options.getArray("actions");
+            Bundle actions = new Bundle();
+            if(acts != null) {
```

Comment:
This does not seem to work, if I start the background task without an `actions` key in the options, the whole background task throws an Error.

Reply:
```
LOG  Trying to start background service
LOG  Error [Error: null]
```
Here is an example of the log